src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -281,6 +284,71 @@ public Predicate<Person> getCurrentPredicate() {
         return result;
     }
 
+    @Override
+    public ObservableList<Tag> getTagFiltersList() {
+        return tagFilters;
+    }
+
+    @Override
+    public ObservableList<RsvpStatus> getStatusFiltersList() {
+        return statusFilters;
+    }
+
+    @Override
+    public void addTagFilters(Set<Tag> tagFilters) {
+        for (Tag tag : tagFilters) {
+            if (!this.tagFilters.contains(tag)) {
+                this.tagFilters.add(tag);
+            }
+        }
+    }
+
+    @Override
+    public void addStatusFilters(Set<RsvpStatus> statusFilters) {
+        for (RsvpStatus status : statusFilters) {
+            if (!this.statusFilters.contains(status)) {
+                this.statusFilters.add(status);
+            }
+        }
+    }
```

Comment:
The if loops for both methods seems to do the same thing, I suggest extracting it to a helper method which both methods can call. This can improve SLAP and prevent the arrowhead indentation as well.

Reply:
Will fix this in next iteration